closure/templates/soy_plugin.bzl

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SoyPluginInfo = provider()
+
+def _impl(ctx):
+    files = []
+    classpath = []
+    for dep in ctx.attr.deps:
+        for jar in dep[JavaInfo].transitive_runtime_deps.to_list():
+            classpath.append(jar.path)
+            files.append(jar)
+    return [SoyPluginInfo(
+        module = ctx.attr.module,
+        deps = ctx.attr.deps,
```

Comment:
I wasn't sure if this should be part of the provider; I don't need it for anything, but I could imagine it being good practice to export the properties for other custom use cases.

Reply:
I suggest making this ⬇️ and compute the classpath/files in `closure_js_template_library` as you did before. Computing the path of the required jars here sounds dangerous if the plugin is defined in workspace `a` and used in workspace `b`.

```python
SoyPluginInfo(
    generator = struct(
        module = ctx.attr.module,
        deps = depset(direct=[dep[JavaInfo] for dep in ctx.attr.deps]),
    ),
)
```

This will make it easier to include the js/java/python runtime for the function in the future (but let's add runtime support in a follow-up CL 😄)

Reply:
Makes sense, updated.